modules/nf-core/fastcov/tests/main.nf.test

```diff
@@ -0,0 +1,58 @@
+nextflow_process {
+
+    name "Test Process FASTCOV"
+    script "../main.nf"
+    process "FASTCOV"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "fastcov"
+
+    test("fastcov single input bam") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [ file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true) ],
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success }
```

Comment:
Maybe you can add some more assertions here. At least the version file should be checked like 
```suggestion
                { assert process.success },
                { assert snapshot(process.out.version).match("version") }
```

Reply:
https://nf-co.re/docs/contributing/tutorials/nf-test_assertions